src/tests/example.spec.js

```diff
@@ -114,4 +114,56 @@ test.describe('Saucedemo app basic tests', () => {
 
         expect(actualSortedResultsZa).toEqual(expectedSortedResultZa);
     });
+
+    test('check Name, Description, and Price values of several added random products in the Shopping Cart', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        // get all the elements of the products
+        const products = await app.inventory.product;
+        const productCount = await products.count();
+
+        // generate three unique indexes
+        const randomIndexes = new Set();
```

Comment:
It's a great approach! But maybe, to avoid the risk of a loop, you can add a check for when there are less than 3 products and adjust the logic accordingly:
const uniqueIndexesCount = Math.min(3, productCount);
while (randomIndexes.size < uniqueIndexesCount) {
    const randomIndex = Math.floor(Math.random() * productCount);
    randomIndexes.add(randomIndex);
}